README.md

```diff
@@ -50,3 +66,14 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Running the Tests
+
+Install test libraries:
+
+Install webdriver: https://chromedriver.chromium.org/getting-started
```

Comment:
In addition to chromedriver, they need the supported version of Chrome installed.